src/EnergyPlus/HeatBalanceMovableInsulation.cc

```diff
@@ -140,7 +140,8 @@ namespace HeatBalanceMovableInsulation {
 		// (I)BLAST legacy routine OMVINS
 
 		// USE STATEMENTS:
-		// na
+		using DataHeatBalance::WindowGlass;
```

Comment:
@Myoldmopar I'll take care of these other comments as soon as I can.  Not sure how a merge conflict got introduced, but glad you know what it is and how to resolve it.